src/stylus/elements/_code.styl

```diff
@@ -13,8 +13,7 @@ code
   &:before
```

Comment:
these styles (`:after`, `:before`) should not be applied to `kbd`

Reply:
OK, I do not know if style `:after`  and `:before` should be applied to `kdb`.
But this produce the same code see [https://unpkg.com/vuetify@0.16.7/dist/vuetify.css](https://unpkg.com/vuetify@0.16.7/dist/vuetify.css).
It is the role of `@extend`

Reply:
You're right, sorry, my bad